itou/www/job_seekers_views/views.py

```diff
@@ -126,10 +126,26 @@ def __init__(self):
     def setup(self, request, *args, **kwargs):
         super().setup(request, *args, **kwargs)
         if self.test_func():
+            job_seekers_created_by_user = User.objects.filter(created_by=request.user).values_list("id", flat=True)
+
+            if request.current_organization:
+                job_seekers_created_by_orga = JobSeekerProfile.objects.filter(
+                    created_by_prescriber_organization=request.current_organization,
+                ).values_list("user_id", flat=True)
+                job_seekers_applications = JobApplication.objects.filter(
+                    (Q(sender=request.user) & Q(sender_prescriber_organization__isnull=True))
+                    | Q(sender_prescriber_organization=request.current_organization),
+                ).values_list("job_seeker_id", flat=True)
+            else:
+                job_seekers_created_by_orga = User.objects.none()
+                job_seekers_applications = JobApplication.objects.filter(sender=request.user).values_list(
+                    "job_seeker_id", flat=True
+                )
+            self.job_seekers_ids = job_seekers_created_by_user.union(
+                job_seekers_created_by_orga, job_seekers_applications
+            )
```

Comment:
J'hésite à suggérer:
```suggestion
            self.job_seekers_ids = list(job_seekers_created_by_user.union(
                job_seekers_created_by_orga, job_seekers_applications
            ))
```
Ça ferait une requête supplémentaire mais permettrait d'avoir ensuite des requêtes simples.

Après même sans ce `list()`, les perfs sont bien meilleures qu'actuellement (~5ms vs ~3500ms :grimacing: ).

Reply:
Merci d'avoir fait les tests de performance en parallèle de cette PR :pray: